parallel_gripper_controller/include/parallel_gripper_controller/parallel_gripper_action_controller_impl.hpp

```diff
@@ -378,6 +385,9 @@ controller_interface::CallbackReturn GripperActionController::on_activate(
   pre_alloc_result_->reached_goal = false;
   pre_alloc_result_->stalled = false;
 
+  // Initialize last movement time
+  last_movement_time_.set(rclcpp::Time(0, 0, RCL_ROS_TIME));
```

Comment:
on_activate is in the RT loop, we should use try_set here, too.
maybe we have to think what happens if try_set fails.

Reply:
Oh I see, I was under the impression that `on_activate` runs in non-RT and only `update()` is RT. I will change this.

If `last_movement_time_.try_set()`  fails during initialization in `on_activate()`, it's a serious issue that should be handled properly. I propose returning `controller_interface::CallbackReturn::ERROR`. What are your thoughts @christophfroehlich ?